lib/datasource/remote/api/balance_repository.dart

```diff
@@ -5,11 +5,7 @@ import 'package:seeds/datasource/remote/model/balance_model.dart';
 import 'package:seeds/datasource/remote/model/token_model.dart';
 
 class BalanceRepository extends NetworkRepository {
```

Comment:
removed the convenience method in order to make sure I change all the places this was being used, and choose the correct token - either hard-coded Seeds token, example for invite, plant, etc, or the currently selected token (send/receive)

Reply:
i would refactor TokenModel out and just pass in what you need from those models. 
The data later doesnt need details of the domain models.